system-tests/src/tests/reset-exercises.spec.ts

```
@@ -0,0 +1,173 @@
+
+test.beforeEach(async ({ browser }) => {
+  context1 = await browser.newContext({ storageState: "src/states/student1@example.com.json" })
+
+  context3 = await browser.newContext({ storageState: "src/states/admin@example.com.json" })
+})
+
+test.afterEach(async () => {
+  await context1.close()
+  await context3.close()
+})
+test("Can manually reset exercises", async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)